### PR TITLE
Simple error readability fix

### PR DESCRIPTION
**Pull request by Glatrix**
Pipe Operator |

### CODE REVIEW

**Comment by ooxi** [comment]
I'm not sure about this. The error message is about the `operator|`, thus I disagree with removing the space between `operator` and `|`.

I'm indifferent to changing *special* to *pipe*

File: common/check_internal.h [hunk 1]
```diff
@@ -37,7 +37,7 @@ class ExitingStream {
   [[noreturn]] ~ExitingStream() {
     llvm_unreachable(
         "Exiting streams should only be constructed by check.h macros that "
-        "ensure the special operator| exits the program prior to their "
+        "ensure the pipe operator | exits the program prior to their "
```
Comment (geoffromer) [inline common/check_internal.h]: This feels a little redundant. How about this instead?

```suggestion
        "ensure the `|` operator exits the program prior to their "
```

**Comment by geoffromer** [comment]
In case you didn't see the message above, you'll need to sign the CLA before we can merge this.

**Comment by Glatrix** [comment]
> In case you didn't see the message above, you'll need to sign the CLA before we can merge this.

I did sign it

File: common/check_internal.h (hunk 1 above)
Comment (Glatrix) [reply common/check_internal.h]: Yeah that also works.

**Comment by jonmeow** [comment]
Sorry, but I'm going to close this PR. This comment refers specifically to the `operator|` on same class. While I appreciate the comment isn't for everyone, I think the comment is clearer as-is and we shouldn't invest effort into changing it.

**Pull request closed without merging**